docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -10,25 +10,40 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.            m = f"Error during WPS job submission (status code: [{_resp.status_code}]) :\n{json_dump(resp.text)}"
</pre><p>Dumped text is not from the correct <code>_resp</code> reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>